HDPS/src/private/PluginManager.cpp

```diff
@@ -632,11 +632,16 @@ void PluginManager::fromVariantMap(const QVariantMap& variantMap)
         if (!_pluginFactories.contains(usedPlugin.toString()))
             missingPluginKinds << usedPlugin.toString();
 
-    if (!missingPluginKinds.isEmpty())
-        throw std::runtime_error(QString("One or more plugins are not available: %1").arg(missingPluginKinds.join(", ")).toLocal8Bit());
-
     if (variantMap.contains("LoadedAnalyses"))
     {
+        for (const auto& loadedAnalysis : variantMap["LoadedAnalyses"].toList())
+            missingPluginKinds.removeAll(loadedAnalysis.toMap()["Kind"].toString());
```

Comment:
If an analysis plugin was used but not saved, we need to load it again and it is not missing